services/app-api/handlers/reports/create.ts

```diff
@@ -59,30 +61,30 @@ export const createReport = handler(
     }
 
     const { state, reportType } = event.pathParameters;
+
+    // If that string they passed in the state paramater isn't a real US State, return No_Key
```

Comment:
Could also remove this comment, I think that the code is easily understandable here

Reply:
Will do!